container/Dockerfile

```diff
@@ -37,19 +36,11 @@ RUN echo -e 'startup_message off \nhardstatus on \nhardstatus alwayslastline \nv
 # Setup vim 
 RUN echo -e 'syntax enable \nset tabstop=4 \nset number \nset hlsearch \nset incsearch' > ${HOME}/.vimrc
 
-# Setup spark/scala/python/rest
+# Setup rest
```

Comment:
Can you move this comment to above line 42, it's more appropriate there.

Reply:
yes